### DISCUSSION TITLE
Change value of column based on value of another column

### DISCUSSION THREAD

**Discussion by humcasma** [vaexio/vaex#1450] [General]
I would like to make a selection of rows, modify the values of some columns on that selection and get the changes applied on the original dataframe. E.g., set column B to 1 on all rows where column A is True.
I can do a selection and make changes to that selection, but how can I get them ported to the original dataframe?

**Comment by JovanVeljanoski** [comment]
Hi, you want to use the `where` method.  Here is an example

```python
import vaex
import vaex.ml
df = vaex.ml.datasets.load_titanic()
df.func.where(df.embarked == 'S', 'Southampton', df.embarked)
```

The where syntax is `where("if", "then", "else")`. I hope this helps!

**Reply by humcasma** [reply]
Yes! This is what I was looking for, but I could not find it in the docs. Thanks!
